src/collections/_documentation/sdks/javascript/errors/context.md

```diff
@@ -0,0 +1,201 @@
+---
+title: Event Context
+excerpt: ""
+---
+
+{%- capture errors_configure_tags -%}
+Most SDKs generally support configuring tags by configuring the scope:
+
+```js
+Sentry.setTag("page_locale", "de-at");
+```
+
+{%- endcapture -%}
+
+{%- capture errors_release_name -%}
+
+For JavaScript developers, a release is also used for applying source maps to minified JavaScript to view original, untransformed source code.
+
+{%- endcapture -%}
+
+{%- capture __errors_set_level -%}
+
+To set the level out of scope, you can call `captureMessage()` per event:
+
+```js
+Sentry.captureMessage('this is a debug message', 'debug');
+```
+
+To set the level within scope, you can call `setLevel()`:
+
+```js
+Sentry.configureScope(function(scope) { 
+   scope.setLevel(Sentry.Severity.Warning);
+});
+```
+
+or per event:
+
+```js
+Sentry.withScope(function(scope) { 
+   scope.setLevel("info"); Sentry.captureException(error);
+});
+```
+{%- endcapture -%}
+
+{%- capture __errors_capture_user -%}
+
+```js
+Sentry.setUser({"email": ["john.doe@example.com](mailto:%22john.doe@example.com)"});
+```
+
+{%- endcapture -%}
+
+{%- capture __errors_grouping_example -%}
+
+```js
+Sentry.configureScope(function(scope) {
```

Comment:
This seems odd that some examples use `configureScope` and some direct `.setX` calls. We don't have `setX` for everything, but it'd be nice to state that we expose only the specific set of `setX` functions because they are used in the majority of cases etc.

Reply:
Could you provide a better example here? I pulled this from other, existing content, which could mean that this example is simply old and needs updating. Thanks.

Reply:
Example is completely valid. It's just that for user, we do `Sentry.setUser`, where
```js
Sentry.configureScope(function(scope) {
  scope.setUser({"email": ["john.doe@example.com](mailto:%22john.doe@example.com)"});
});
```
would work as well. It's just not symmetrical with `setFingerprint` for example. The problem is that there's no `Sentry.setFingerprint` equivalent, so I think we can leave it as is for now.

Reply:
I'll leave it then. Thanks.